Sources/Common/Util/Log.swift

```diff
@@ -98,3 +98,20 @@ public class ConsoleLogger: Logger {
     public func error(_ message: String) {}
     #endif
 }
+
+extension CioLogLevel {
+    static func getLogLevel(for value: String) -> CioLogLevel? {
+        switch value.lowercased() {
+        case "none":
+            return CioLogLevel.none
+        case "error":
+            return .error
+        case "info":
+            return .info
+        case "debug":
+            return .debug
```

Comment:
```suggestion
        case CioLogLevel.none.rawValue:
            return CioLogLevel.none
        case CioLogLevel.error.rawValue:
            return .error
        case CioLogLevel.info.rawValue:
            return .info
        case CioLogLevel.debug.rawValue:
            return .debug
```

Since `CioLogLevel` now uses `: String`, we can use `rawValue`. This seems more maintainable, but maybe it would be a bad idea? 

The only flaw I see if we might by accident introduce a breaking change into the public API for our SDK wrappers. 

I suggest we add a test function to Common/APITest which makes sure that we keep the hard-coded string values "none", "error", "info", "debug" getting a log level. So we don't, by accident, break logging for SDK wrappers when they configure the SDK.